### ISSUE TITLE
Conversion bug

### ISSUE THREAD

**Issue by busheezy** [babofitos/node-steamid-converter#4]
When converting 64 to string it always starts with "STEAM_0:1" instead of the possible "STEAM_1:1."

**Comment by babofitos** [comment]
My understanding is that this doesn't matter, both should be fine for conversion purposes. Could you produce an id that wouldn't convert? Unless I'm mistaken I don't think it's possible to figure it out from a 64 id regardless.

**Comment by busheezy** [comment]
Well it kind of matters but you're probably right that it isn't able to be figured out.  I'm using another library that lets me easily generate the "new style" and the "old style."  I'm having to check the database for both results.  

Thanks for your time
,Ryan

https://gyazo.com/ee609e81f3ec0549abdf22f36c47da31

**Comment by babofitos** [comment]
Yeah honestly I should just drop the STEAM_0: part altogether when converting instead of making assumptions.
